dan_layer/p2p/proto/consensus.proto

```diff
@@ -74,6 +74,7 @@ message ForeignProposal {
   bytes block_id = 2;
   ForeignProposalState state = 3;
   uint64 mined_at = 4;
```

Comment:
Suggest `proposed_height`. This has already changed in development but I missed this in the proto file